Index.html

```diff
@@ -33,6 +34,154 @@ <h2 class="subtitulo">I&apos;am a Software Developer</h2>
+            </div>
+              <button><a class="boton2">See Project</a></button> 
+            </div>
+    
+            <div class="projects">
+              <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+              <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <div class="list_programs2">
+                <ul>
+                  <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                  <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                  <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                </ul>
+              </div>
+                <button><a class="boton2">See Project</a></button> 
+              </div>
+
+              <div class="projects">
+                <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+                <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="list_programs2">
+                  <ul>
+                    <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                    <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                    <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                  </ul>
+                </div>
+                  <button><a class="boton2">See Project</a></button> 
+                </div>
+        
+                <div class="projects">
+                  <h1 class="tituloPAPD">Profesional Art Printing Data</h1>
+                  <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="list_programs2">
+                    <ul>
+                      <li<a><img class="htm"src="./images/html2.svg"></a></li> 
+                      <li<a><img class="bot"src="./images/boot2.svg"></a></li>
+                      <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+                    </ul>
+                  </div>
+                    <button><a class="boton2">See Project</a></button> 
+                  </div>
+
+    </section>
+
+  <section class="page3">
+    <h1 class="about">About me</h1>
+    <p class="parrafoAM">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <button><a class="boton3">Get My Resume</a></button> 
+    <img class="vector2"src="./images/Vector.svg">
```

Comment:
- [ ] I noticed the button in your about section looks different from the design template. Also, the spacings in the section are small compared to the design template. Kindly fix that please as the project requirements demanded that you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] Please move the background image of this section to be at the appropriate place. I noticed yours is at the bottom of the section.
### Your design
![2](https://user-images.githubusercontent.com/94551484/198817662-9ea518c0-9f43-4449-a833-9ade48c61bbb.png)

### Figma template
![f2](https://user-images.githubusercontent.com/94551484/198817668-3fb0f01b-49ba-40d8-b832-9914a7c52a68.png)